bundles/org.openhab.binding.revogismartstripcontrol/pom.xml

```
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.3-SNAPSHOT</version>
```

We just had a new release today so this needs to be changed, yet again...sorry about that.
```suggestion
    <version>2.5.4-SNAPSHOT</version>
```

---

ok, here we go :)

---

travis build succeeded :)